src/pages/products/Filters.jsx

```diff
@@ -0,0 +1,155 @@
+import { useProductFilter } from '../../context/product-context'
```

Comment:
Have an index.js file in the context folder in there import all context in an index.js and then export so while importing it will look like import "../context"

Reply:
Thanks implemented it